tests/unit/client.py

```diff
@@ -602,75 +602,75 @@ class Test_user:
 
     def test_user_create(self):
         """ Test user creation. """
-        assert C.user.create('billy', 'pass1234', 'admin') is None
-        assert C.user.create('bobby', 'pass1234', 'regular') is None
+        assert C.user.create('billy', 'pass1234', True) is None
+        assert C.user.create('bobby', 'pass1234', False) is None
```

Comment:
It would probably be a bit more readable to pass this argument by name, e.g `is_admin=True`, now that the value is less self-describing.

Reply:
good point, will add that in the next commit.